microsoft/testsuites/hpc/infinibandsuite.py

```diff
@@ -31,6 +33,30 @@
     """,
 )
 class InfinibandSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. List all available network interfaces
+        2. Check if InfiniBand cards are present
+        3. Ensure InfiniBand cards are named starting with "ib0"
+        """,
+        priority=2,
+        requirement=simple_requirement(supported_features=[Infiniband]),
+    )
+    def verify_ib_naming(self, log: Logger, node: Node) -> None:
+        ib_interfaces = node.features[Infiniband].get_ib_interfaces()
+        ib_first_device_name = ib_interfaces[0].ib_device_name
+
+        r = re.compile("ib[0-9]")
```

Comment:
[nit] Improve variable naming and provide an example in comments above this line